src/containers/Forms/Student/StudentForm.js

```diff
@@ -34,12 +35,17 @@ const languages = [
 const StudentForm = ({ title, submitData, selectedStudent, id }) => {
   const [student, setStudent] = useState(selectedStudent);
   const { streams } = useSelector((state) => state.streams);
+  const { groups } = useSelector((state) => state.groups);
   const dispatch = useDispatch();
 
   useEffect(() => {
     dispatch(fetchStreams());
   }, []);
```

Comment:
```suggestion
  useEffect(() => {
    dispatch(fetchStreams());
    dispatch(fetchGroups());
  }, []);   
```
А что если объединить два useEffect-а?